src/static/bitbucket/components/Warnings.tsx

```diff
@@ -1,14 +1,24 @@
+import listStyles from './styles/listStyles';
+
 type WarningProps = {
   warnings: string[];
 };
 
+import styled from 'styled-components';
```

Comment:
why are we mixing styled-components?

Reply:
Ah right, I missed converting this one to emotion. Will do.